search/src/main/java/artic/edu/search/SearchFragment.kt

```diff
@@ -19,4 +23,17 @@ class SearchFragment : BaseViewModelFragment<SearchViewModel>() {
 
     override val customToolbarColorResource: Int
         get() = R.color.greyText
+
+    override fun setupBindings(viewModel: SearchViewModel) {
+        super.setupBindings(viewModel)
+
+        toolbar!!.searchEditText
```

Comment:
should probably use toolbar?.let OR use the toolbar from kotlinx

Reply:
I couldn't get the kotlinx one to import, didn't want to diagnose. I'll leave it for now.